c++/src/kj/async-test.c++

```diff
@@ -509,6 +509,47 @@ TEST(Async, Fork) {
   EXPECT_EQ(789, branch2.wait(waitScope));
 }
 
+TEST(Async, ForkMaybeOwn) {
+  struct MyType {
+    int value = 123;
+
+    Own<MyType> addRef() const {
+      return kj::heap<MyType>(MyType{ .value = 456, });
```

Comment:
Build failure because MSVC doesn't support designated initializers until C++20.

```suggestion
      return kj::heap<MyType>(MyType{456});
```

Reply:
Resolved by switching to use ForkRef as the base test.